packages/inter-protocol/src/reserve/assetReserveKit.js

```diff
@@ -166,6 +198,67 @@ export const prepareAssetReserveKit = async (
           facets.helper.writeMetrics();
         },
       },
+      withdrawalFacet: {
+        Withdraw() {
+          /** @type {OfferHandler<Promise<string>>} */
+          const handler = async seat => {
```

Comment:
This handler is not durable. `Withdraw()` invitations don't need to survive upgrade, right?

As long as their continuing invitation survives, the client can just ask for a new one.